docs/examples/search_vector_similarity_examples.ipynb

```diff
@@ -19,39 +23,126 @@
        "b'OK'"
       ]
      },
-     "execution_count": 1,
+     "execution_count": 2,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
     "import redis\n",
-    "from redis.commands.search.field import VectorField\n",
+    "\n",
+    "from redis.commands.search.indexDefinition import (\n",
+    "    IndexDefinition,\n",
+    "    IndexType\n",
+    ")\n",
     "from redis.commands.search.query import Query\n",
```

Comment:
Can we sort the imports - the rest of the project relies on isort style (sorted) imports. It would be nice to share the consistency with our docs.

Reply:
Done!